<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VisualizationPanelWidget</class>
 <widget class="QWidget" name="VisualizationPanelWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>673</width>
    <height>151</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="vizIntensityMinMaxResetButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>25</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="vizIntensityMinSpinBox">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>7</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QSlider" name="vizIntensityMaxSlider">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>37</y>
     <width>160</width>
     <height>22</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="minimum">
    <number>-50</number>
   </property>
   <property name="maximum">
    <number>150</number>
   </property>
   <property name="value">
    <number>99</number>
   </property>
   <property name="tracking">
    <bool>true</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="invertedControls">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="vizIntensityMaxSpinBox">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>37</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QSlider" name="vizIntensityMinSlider">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>7</y>
     <width>160</width>
     <height>22</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="minimum">
    <number>-50</number>
   </property>
   <property name="maximum">
    <number>150</number>
   </property>
   <property name="sliderPosition">
    <number>2</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="invertedAppearance">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="vizUpdate2DOverlayButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Update 2D Overlay</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="vizAutoUpdate2DOverlayCheckBox">
   <property name="geometry">
    <rect>
     <x>141</x>
     <y>73</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Auto Update 2D Overlay</string>
   </property>
  </widget>
  <widget class="QPushButton" name="vizUpdate3DSceneButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>107</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Update 3D Scene</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="vizAutoUpdate3DSceneCheckBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>110</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Auto Update 3D Scene</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="vizIntensityMinLabel">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>7</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Intensity Min</string>
   </property>
  </widget>
  <widget class="QLabel" name="vizIntensityMaxLabel">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>37</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Intensity Max</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
